apps/mudita-center-e2e/src/specs/news/news-check-offline.e2e.ts

```diff
@@ -0,0 +1,106 @@
+import exp from "constants"
```

Comment:
unused

Reply:
it was used in online mode test so I just copied it